force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -68,7 +76,11 @@ public with sharing class VariablesDataTypesOperators {
      * @return The quotient of the division, or 0 if the denominator is zero.
      */
     public static Decimal division(Integer a, Integer b) {
-        return null; // Replace null with the variable you used to store the result
+        If(b != 0){
+            Integer result = a/b;
+            return result;
+        }
+        return 0; // Replace null with the variable you used to store the result
```

Comment:
Try using the ternary operator.

Reply:
Something like this?

b != 0 ? return a / b : return 0;